NewHorizons/Assets/translations/portuguese_br.json

```diff
@@ -1,6 +1,69 @@
 {
-  "$schema": "https://raw.githubusercontent.com/xen-42/outer-wilds-new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "$schema": "https://raw.githubusercontent.com/outer-wilds-new-horizons/new-horizons/main/NewHorizons/Schemas/translation_schema.json",
+  "DialogueDictionary": {
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_1": "Sua nave agora está equipada com uma unidade de translocação!",
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_2": "Você pode usar a nova aba \"Modo Interestelar\" no diário de bordo de sua nave para ajustar o piloto automático rumo a outro sistema solar.",
+    "NEW_HORIZONS_WARP_DRIVE_DIALOGUE_3": "Então é só acionar o piloto automático e relaxar!"
+  },
   "UIDictionary": {
-    "Vessel": "Hospedeiro"
+    "INTERSTELLAR_MODE": "Modo Interestelar",
+    "FREQ_STATUE": "Estátua Nomai",
+    "FREQ_WARP_CORE": "Fluxo Anti-Gravitacional",
+    "FREQ_UNKNOWN": "???",
+    "ENGAGE_WARP_PROMPT": "Acionar translocação a {0}",
+    "WARP_LOCKED": "PILOTO AUTOMÁTICO AJUSTADO PARA:\n{0}",
+    "LOCK_AUTOPILOT_WARP": "Ajustar Piloto Automático para sistema solar",
+    "RICH_PRESENCE_EXPLORING": "Explorando {0}.",
+    "RICH_PRESENCE_WARPING": "Translocando para {0}.",
+    "OUTDATED_VERSION_WARNING": "AVISO\n\nO mod New Horizons funciona apenas na versão {0} ou superior. Você está usando a versão {1}.\n\nAtualize o Outer Wilds ou desinstale NH.",
+    "JSON_FAILED_TO_LOAD": "Arquivo(s) inválido(s): {0}",
+    "DEBUG_RAYCAST": "Raycast",
+    "DEBUG_PLACE": "Posicionar objeto",
+    "DEBUG_PLACE_TEXT": "Colocar texto Nomai",
+    "DEBUG_UNDO": "Desfazer",
+    "DEBUG_REDO": "Refazer",
+    "Vessel": "Hospedeiro",
+    "DLC_REQUIRED": "AVISO\n\nVocê tem addons (ex. {0}) instalados que requerem a DLC, mas a mesma não está habilitada.\n\nSuas mods podem não funcionar como esperado."
+  },
+  "OtherDictionary": {
+    "NOMAI_SHUTTLE_COMPUTER": "A <![CDATA[<color=orange>exploradora</color>]]> está repousando por hora em: <![CDATA[<color=lightblue>{0}</color>]]>."
+  },
+  "AchievementTranslations": {
```

Comment:
In the achievements I just changed them to fit better with steam usual descriptions

Reply:
Agreed, I don't really have steam in Portuguese so I am not used to see how they translate achievements after all. I was unsure which "verbal time" to apply to the achievements.

Reply:
Fair, I believe they usually word as the instructions to get the achievement. Like how it is in english, ie "Fight X monsters", "Fly for Y seconds"